composer.json

```diff
@@ -28,6 +32,15 @@
         }
     },
     "require-dev": {
-        "sllh/php-cs-fixer-styleci-bridge": "^2.1"
+        "matthiasnoback/symfony-config-test": "^3.0",
+        "php-http/guzzle6-adapter": "^1.1",
```

Comment:
Have you test about http calls? If not, this can be removed for now.

Reply:
If I remove this package, travis return an error 
```
- The requested package php-http/client-implementation could not be found in any version, there may be a typo in the package name.
```